Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -96,6 +96,34 @@ final class CacheAdvanceTests: XCTestCase {
         XCTAssertEqual(messages, [])
     }
 
+  func test_messages_throwsFileCorruptedWhenOffsetInFileAtEndOfNewsetMessageOutOfSync() throws {
+      let header = try CacheHeaderHandle(
+        forReadingFrom: testFileLocation,
+        maximumBytes: 10_000,
```

Comment:
this number is effectively random. Doesn't matter what it is.

Reply:
let's add this to a local constant called `randomHighValue`, that way we document this via the property name.

Reply:
2470113